schema/dereferenced-release-schema.json

```
@@ -1012,7 +1012,7 @@
         },
         "documents": {
           "title": "Documents",
-          "description": "A list of documents related to the planning process.",
+          "description": "Documents related to the planning stage (for example notices, needs assessments and market studies).",
```

Add a comma after "for example" in all cases.

```suggestion
          "description": "Documents related to the planning stage (for example, notices, needs assessments and market studies).",
```

---

Added in https://github.com/open-contracting/standard/pull/1229/commits/c351ca4cbb9df2edaaebda33100ede21ee1069db